client/src/components/confirmation.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Styles from './confirmation.module.css';
+import RestaurantInfo from './restaurantInfo.jsx';
+
+class Confirmation extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            isChecked: false
+        }
+    }
+    handleCheckboxChange() {
+        this.setState({
+            isChecked: !this.state.isChecked
```

Comment:
consider checking out whether writing it with state as an argument would be worthwhile, ex . this.setState((state)=>{ return {isChecked: !state.isChecked}}).  Probably fine as is though